examples/typescript/aws-s3-static-website/__tests__/main-test.ts

```diff
@@ -0,0 +1,87 @@
+import "cdktf/lib/testing/adapters/jest"; // Load types for expect matchers
```

Comment:
HashiCorp has been rolling out a new policy to stay compliant with our MPL-2.0 license which means all our files (with a few exceptions) need a copyright header at the top:

```
// Copyright (c) HashiCorp, Inc
// SPDX-License-Identifier: MPL-2.0
```

You can either add these manually or use HashiCorp's [copywrite](https://github.com/hashicorp/copywrite) tool (recommended, as it's configured to skip the exceptions).